src/components/index.ts

```diff
@@ -31,6 +31,9 @@ export * from './ghost';
 export { default as Header } from './header';
 export * from './header';
 
+export { default as InfiniteScroll } from './infinite-scroll';
```

Comment:
FYI, we may soon be able to use prefer named exports in Typescript files. There is an RFC, [Coupled Migrations of Typescript and Named Exports in BUIE](https://box.slack.com/archives/CG8KSE9DJ/p1645049343055679), which I believe might be taken up by web-arch-group on Monday, so that'd be cool. For now, still default exports.

Reply:
Yes, defaults are a bad idea imo (and I think there's a general consensus about it now in the community) so I totally agree with that